origin.spec

```diff
@@ -413,6 +418,10 @@ fi
 
 %post node
 %systemd_post %{name}-node.service
+# If accounting is not currently enabled systemd reexec
+if [[ `systemctl show mananager | grep -q -e CPUAccounting=no -e MemoryAccounting=no` == 0 ]]; then
```

Comment:
Yeah, looks look `systemctl show bogus` works too though i want to verify that's not just unexpected behavior.